docs/pages/guides/server-components.mdx

```diff
@@ -4,28 +4,26 @@ sidebar_title: React Server Components
 description: Learn about rendering React components on the server in Expo.
 ---
 
-import { Terminal } from '~/ui/components/Snippet';
 
 > Experimentally available in **SDK 52 and above**. This is a very early preview and not ready for production or wide use yet. This initial version is meant to help library authors understand universal React Server Components and add support in their libraries.
 
-Expo Router enables support for [React Server Components](https://react.dev/reference/rsc/server-components) on all platforms.
 
-React Server Components enable a number of exciting capabilities including:
+React Server Components enable a number of exciting capabilities, including:
 
 - Data fetching with async components and React Suspense.
 - Working with secrets and server-side APIs.
 - Server-side rendering (SSR) for SEO and performance.
 - Build-time rendering to remove unused JS code.
 
-This is an early preview of a feature that will be enabled by default in Expo Router.
+Expo Router enables support for [React Server Components](https://react.dev/reference/rsc/server-components) on all platforms. This is an early preview of a feature that will be enabled by default in Expo Router.
 
 ## Usage
```

Comment:
```suggestion
## Prerequisites

Your project must use Expo Router and React Native new architecture. New Architecture is not supported in Expo Go, so you'll need to [create a development build](/develop/development-builds/introduction/) or create a web app.

As of SDK 52, there is no strategy for migrating existing projects. We recommend starting with a new project.

## Usage
```

Let's add a prerequisite section here to highlight the requirements of using server components.
Using the web doesn't provide much information as to what it means. I think you meant here to be a web app.